lib/ProductOpener/Display.pm

```diff
@@ -10578,7 +10578,7 @@ sub display_rev_info {
 	};
 
 	my $html;
-	process_template('display_rev_info.tt.html', $template_data_ref, \$html) || return 'template error: ' . $tt->error();
+	process_template('web/pages/rev_info/display_rev_info.tt.html', $template_data_ref, \$html) || return 'template error: ' . $tt->error();
```

Comment:
this one is part of the product page in fact, it should be in templates/web/pages/product/includes/